src/common/routes.ts

```diff
@@ -40,6 +40,7 @@ export const routes = {
     index: '/campaigns',
     create: '/campaigns/create',
     viewCampaignBySlug: (slug: string) => `/campaigns/${slug}`,
+    viewCampaignById: (id: string) => `/campaigns/${id}`,
```

Comment:
This route doesn't seem to exist in `/pages` folder

Please replace or remove it